docs/manuals/core/config/data-node-config.md

```diff
@@ -414,16 +414,31 @@ In this example, we configure an in_memory data node with the id "date", the sco
 
 # JSON
 
-An `JSONDataNode^` is a type of data node used to model JSON
+A `JSONDataNode^` is a type of data node used to model JSON
 file data. To add a new _JSON_ data node configuration, the `Config.configure_json_data_node_node()^` method can be used.
-In addition to the generic parameters described in the previous section
-[Data node configuration](data-node-config.md), a mandatory and an optional parameters can be provided.
+In addition to the generic parameters described in
+[Data node configuration](data-node-config.md) section, a mandatory and two optional parameters can be provided.
 
-- The _**default_path**_ is a mandatory parameter that represents the JSON file path used by Taipy to read and write the data.
+- The _**default_path**_ is a mandatory parameter that represents the JSON file path used by Taipy to read and write data.
 
 - The _**encoder** and _**decoder**_ parameters are optional parameters that represent the encoder (json.JSONEncoder) and decoder
-  (json.JSONDecoder) used to serialize and deserialize the JSON data.
+(json.JSONDecoder) used to serialize and deserialize JSON data.
+
+```python linenums="1"
+from taipy import Config
 
+hist_temp_cfg = Config.configure_json_data_node(
+    id="historical_temperature",
+    default_path="path/hist_temp.json",
+)
+```
+
+In this example, we configure a JSON data node. The _id_ argument is "historical_temperature". Its _scope_ is
+`SCENARIO` (default value), and the path is the file hist_temp.json.
+
+Without specific _**encoder** and _**decoder**_ parameters, `hist_temp_cfg` will use the `DefaultJSONEncoder` and
+`DefaultJSONDecoder` provided by Taipy, which have the capability to encode and decode Python [`enum.Enum`](https://docs.python.org/3/library/enum.html), [`datetime.datetime`](https://docs.python.org/3/library/datetime.html#datetime-objects),
+and [dataclass](https://docs.python.org/3/library/dataclasses.html) object.
```

Comment:
```suggestion
Without specific _**encoder** and _**decoder**_ parameters, `hist_temp_cfg` will use default encoder and decoder provided by Taipy, which have the capability to encode and decode Python [`enum.Enum`](https://docs.python.org/3/library/enum.html), [`datetime.datetime`](https://docs.python.org/3/library/datetime.html#datetime-objects),
and [dataclass](https://docs.python.org/3/library/dataclasses.html) object.
```

I removed the reference to the DefaultJSONEncoder class because it has been made private.